desktop/cmp/input/CodeInput.js

```diff
@@ -300,6 +430,86 @@ export class CodeInput extends HoistInput {
         }
     }
 
+    findAll = () => {
+        this.clearSearchResults();
+        this.cursor = this.editor.getSearchCursor(this.query, 0, true);
+
+        const {cursor, editor, selectedMatches} = this;
+        while (cursor.findNext()) {
+            const anchor = cursor.from(),
+                head = cursor.to();
+            selectedMatches.push({
+                anchor,
+                head,
+                textMarker: editor.markText(anchor, head, {className: 'selection-highlight'})
```

Comment:
Here I would use the class `xh-code-input--highlight` or similar - key part being to put that prefix in.  We're not worried about breaking other custom app styles - since we nest our rules under `.xh-code-input` - but you could imagine an app defining a `selection-highlight` class for some other, completely unrelated reason and breaking _our_ styling here.